src/test/java/org/openrewrite/staticanalysis/EqualsToContentEqualsTest.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2021 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.staticanalysis;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+public class EqualsToContentEqualsTest implements RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec
+          .parser(JavaParser.fromJavaVersion())
+          .recipe(new EqualsToContentEquals());
+    }
+
+    @Test
+    public void replaceStringBuilder() {
```

Comment:
Try to always annotate one representative test with `@DocumentExample`, such that we get code blocks to show up in the docs, such as for instance in: https://docs.openrewrite.org/recipes/java/testing/cleanup/removetestprefix
```suggestion
    @Test
    @DocumentExample
    public void replaceStringBuilder() {
```